src/CommaSeparatedFloatParser.php

```diff
@@ -11,10 +11,10 @@ protected function parse($value)
         }
         $returnedIntArr = [];
         foreach (explode(',', $value) as $num) {
-            if (!is_numeric($num)) {
+            if (!is_numeric(trim($num))) {
```

Comment:
But shouldn't e.g.  `1 ` or ` 1` be invalid input? IMO the client should be forced to send valid input only (atleast for now to be consistent with the existing `$this->param(..)->int()` because there is also no trimming)